src/main/antlr3/com/joestelmach/natty/generated/DateWalker.g

```diff
@@ -22,7 +23,14 @@ options {
     throw e;
   }
 
+  public void setBaseDate(java.util.Date referenceDate) {
```

Comment:
Thanks Jack, but I think we should use the term 'referenceDate' everywhere, including these method names. (And I fully realize the CalendarSource I wrote used the name baseDate :), but referenceDate seems like a better fit to me now)